spec/index.bs

```diff
@@ -1249,17 +1249,9 @@ partial interface mixin WindowOrWorkerGlobalScope {
 </pre>
 
 To the [[HTML5#timer-initialisation-steps|timer initialization steps algorithm]],
-add this step between 7.1 and 7.2:
+change step 8.3.2 as follows:
```

Comment:
In https://html.spec.whatwg.org/#timer-initialisation-steps. that's now 8.3.4. Maybe that changes again?
Sorry for the nit-picking but from experience, specs sometimes stay unintendedly in 'temporary' states. Therefore each patch should not rely on other patches to come.